src/plone/distribution/api/site.py

```diff
@@ -140,14 +140,14 @@ def create(
     context,
     distribution_name: str,
     answers: dict,
-    profile_id: str = _DEFAULT_PROFILE,
+    profile_id: str = DEFAULT_PROFILE,
 ) -> PloneSite:
     """Create a new Plone site using one of the distributions.
 
     :param context: Context where the site will be created.
     :param distribution_name: Name of distribution to be used.
     :param answers: Payload for site creation.
-    :param profile_id: Base profile to be used.
+    :param profile_id: (deprecated) Base profile to be used.
```

Comment:
"deprecated" suggests to me that this should not be used anymore because it the option will be removed in the future, but currently it still works.

```suggestion
    :param profile_id: (deprecated and ignored) Base profile to be used.
```